gamestonk_terminal/cryptocurrency/onchain/onchain_controller.py

```diff
@@ -849,6 +1383,14 @@ def print_help(self):
 
 Whale Alert:
     whales            check crypto wales transactions
+
+BitQuery (note: some commands execution can take > 10 seconds):
```

Comment:
do you think it is possible to add a progress indicator for the commands that take more time?
(`sia` menu has good progress indicators for some of the commands)

Reply:
a progress makes sense if there are multiple calls made, which might not be the case here (but i don't know for sure)

Reply:
Agree with piiq, in case if we would scrape/query multi pages it would be good idea to do that, but in my case it's just a single query, but it's seems that it's expensive for bitquery server. I tried to limit somehow all queries in the way that time to recieve data will be pretty fast (like few seconds)

Reply:
Can we remove that note though?

I feel like a lot of commands have a waiting time, and if we want to notify the user about this should happen at the command level, like a print when we execute it. Rather than being in the menu adding noise.